pkg/cdi/container-edits.go

```diff
@@ -162,6 +172,14 @@ func (e *ContainerEdits) Apply(spec *oci.Spec) error {
 	return nil
 }
 
+func specgenAddLinuxNetDevice(specgen *ocigen.Generator, hostIf string, netDev *oci.LinuxNetDevice) {
+	if specgen == nil || netDev == nil {
+		return
+	}
+	ensureLinuxNetDevices(specgen.Config)
+	specgen.Config.Linux.NetDevices[hostIf] = *netDev
+}
+
```

Comment:
Let's implement this function and `ensureLinuxNetDevices` somewhere "near" each other in the file.

Reply:
That's ok. Maybe we can also contribute that upstream along with the netDevices code?